### PR TITLE
Maint Bar 2.0 [MEGA SOUL ALERT]

### PR DESCRIPTION
**Pull request by thwompa**
# About the pull request

This replaces a vacant space in maint just above the Alpha/Bravo bunks with a small hideout for unscrupulous activities. The old maint bar was removed with the cryo update Bobby made, so this sticks it back in with 10x more soul.

# Explain why it's good for the game

While anybody can take any old closet in maint and use it for whatever they want, the bar intends to serve as the primary hub, and go-to locations for individuals looking to engage in poor behavior in relative isolation. (roleplay center)

# Screenshots & Videos

image

# Changelog

:cl:
maptweak: Converted vacant space in the lower-starboard hull maint area to a secret hideout.
/:cl:

### CODE REVIEW

**Comment by thwompa** [comment]
i fucked the pig

**Comment by ItsVyzo** [comment]
picture

**Comment by thwompa** [comment]
> picture
press da dropdown bayebee

File: maps/map_files/USS_Almayer/USS_Almayer.dmm [hunk 1]
```diff
@@ -71980,6 +72042,19 @@
...
 	},
 /area/almayer/shipboard/weapon_room)
+"sSe" = (
+/obj/structure/sign/poster/blacklight{
+	pixel_y = 30
+	},
+/obj/structure/machinery/light/small{
+	dir = 8
+	},
+/obj/structure/reagent_dispensers/beerkeg/alt_dark,
```
Comment (Agameofscones) [inline maps/map_files/USS_Almayer/USS_Almayer.dmm:72052]: anchored = 1
density = 0
pixel off set it into the corner, reduce the contents to ~10-20%, better yet, write a quick and dirty subtype that changes the contents randomly via proc on init,

**Review by Agameofscones — APPROVED** [review]
yeah lgtm

**Review by harryob — APPROVED** [review]
no code

**Pull request merged**